cli/cli-client/src/main/java/org/infinispan/cli/connection/rest/RestConnection.java

```diff
@@ -247,6 +257,60 @@ public String execute(List<CommandInputLine> commands) throws IOException {
                }
                break;
             }
+            case Backup.CMD: {
+               String container = getActiveContainer().getName();
+               RestCacheManagerClient manager = this.client.cacheManager(container);
+               responseMode = ResponseMode.BODY;
+               String backupName;
+               switch (command.arg(CliCommand.TYPE)) {
+                  case Backup.Create.CMD:
+                     backupName = command.optionOrDefault(Backup.NAME, () -> {
+                        // If the backup name has not been specified generate one based upon the Infinispan version and timestamp
+                        LocalDateTime now = LocalDateTime.now();
+                        return String.format("%s-%tY%2$tm%2$td%2$tH%2$tM%2$tS\n", Version.getBrandName(), now);
+                     });
+
+                     sb.append("Creating backup '").append(backupName).append("'");
```

Comment:
shouldn't this have a line break at the end ?

Reply:
It's unnecessary, as the return value of `execute(commands)` is written via `writeln()`. https://github.com/infinispan/infinispan/blob/fa0528016009c2c3ec9a224b839c8a502110cd0c/cli/cli-client/src/main/java/org/infinispan/cli/impl/ContextImpl.java#L221-L224